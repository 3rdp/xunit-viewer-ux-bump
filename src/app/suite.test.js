import React from 'react'
import Suite from './suite'
import renderer from 'react-test-renderer'

test('renders suite', () => {
  const tree = renderer.create(<Suite
    id='suite-id'
    name='suite-name'
    properties={{
      a: 'b',
      _visible: true
    }}
    active
    time={100}
    tests={{
      testid: {
        visible: true,
        id: 'testid',
        messages: ['message 1', 'message 2'],
        status: 'passed',
        time: 888,
        name: 'Test name',
        suite: 'suite-id'
      }
    }} />).toJSON()
  expect(tree).toMatchSnapshot()
})
