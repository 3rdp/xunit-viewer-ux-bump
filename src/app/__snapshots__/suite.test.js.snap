// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders suite 1`] = `
<div
  className="renderIfVisible "
>
  <div
    className="renderIfVisible-placeholder "
    style={
      Object {
        "height": 300,
      }
    }
  />
</div>
`;
